Automattic/jetpack
# API endpoints: handle updated Requests library

## Proposed changes:

Let's start using the updated library now available in WordPres 6.2.

### Other information:


## Jetpack product discussion

Follow-up of #27981
See #27795

Dev-note: https://make.wordpress.org/core/2023/03/08/requests-library-upgraded-to-2-0-5-in-wordpress-6-2/

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Start with 2 sites, connected to WordPress.com, with the Publicize module active, and connected to a Twitter account:

- One running WordPress 6.1
- One running WordPress 6.2 Beta

We'll test Twitter features to see that things still work., while tailing logs for the 2 sites to watch for any notices.

* Go to Posts > Add New, and add some content
* Add a Twitter block and paste a Twitter URL for a thread, e.g. `https://twitter.com/doctorow/status/1636413428943278081`
* Click on Unroll
* Watch the tweet unroll
* Open the Jetpack sidebar
* Change Publicize options to publish a thread instead of a single tweet.
* Watch the separators added to the post content.
* Click on the "Social Previews" section at the bottom of the sidebar.
* You should see previews under some of the tweets, and you should not see any notices in your logs.
* Now add an image block, and select "Select Image > Google Photos".
* If you have not connected your Google account yet, you should be able to do so now.
* Once you've connected, hit the disconnect button in the top right corner of the modal.
* It should disconnect without generating any notices in your logs.
